tests/glsl-intrinsic/fragment-processing/fragment-processing.slang

```diff
@@ -0,0 +1,143 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL): -target glsl -stage fragment -entry main -allow-glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV): -target spirv -emit-spirv-directly -stage fragment -entry main -allow-glsl
+#version 450
+buffer MyBlockName
+{
+    vec4 fdata[2];
+    ivec4 udata[2];
+    uvec4 idata[2];
+} keepAliveBuffer;
+
+layout(location = 0) out ivec4 outColorActual;
+
+layout (location = 0) in float inDataV1;
+layout (location = 1) in vec2 inDataV2;
+layout (location = 2) in vec3 inDataV3;
+layout (location = 3) in vec4 inDataV4;
+
+bool testFragmentProcessingDerivativeFunctionsScalar()
+{
+// CHECK_SPV-DAG: OpDPdx
+// CHECK_GLSL-DAG: dFdx
+// CHECK_SPV-DAG: OpDPdy
+// CHECK_GLSL-DAG: dFdy
+// CHECK_SPV-DAG: OpDPdxFine
+// CHECK_GLSL-DAG: dFdxFine
+// CHECK_SPV-DAG: OpDPdyFine
+// CHECK_GLSL-DAG: dFdyFine
+// CHECK_SPV-DAG: OpDPdxCoarse
+// CHECK_GLSL-DAG: dFdxCoarse
+// CHECK_SPV-DAG: OpDPdyCoarse
+// CHECK_GLSL-DAG: dFdyCoarse
+// CHECK_SPV-DAG: OpFwidth
+// CHECK_GLSL-DAG: fwidth
+// CHECK_SPV-DAG: OpFwidthFine
+// CHECK_GLSL-DAG: fwidthFine
+// CHECK_SPV-DAG: OpFwidthCoarse
+// CHECK_GLSL-DAG: fwidthCoarse
```

Comment:
Isn't the order deterministic?
Can you remove "-DAG"?

Reply:
you are correct, this should not be `-DAG` (some previous `interpolation` function fixes causes non-deterministic code order, I forgotten to change this back)